### ISSUE TITLE
Missing word "добро"

### ISSUE THREAD

**Issue by mirmozavr** [Harrix/Russian-Nouns#3]
No word "добро"

**Comment by mirmozavr** [comment]
> @mirmozavr 5букв от Тинькофф?)

Да, сделал себе решатель на основе этого репо) Ещё составил список популярных букв. Если использовать КРОАТ и СЕЛИН, то почти всегда на 3й попытке можно определить ответ. Если нет: ОБДУВ :)

**Comment by MRGRD56** [comment]
@mirmozavr 

> Да, сделал себе решатель на основе этого репо)

[То же самое)](https://github.com/MRGRD56/tinkoff-5bukv-solver) А сегодня провалил, ибо этого самого "добро" в списке нет. Побежал писать issue, а тут меня уже опередили))

> Ещё составил список популярных букв. Если использовать КРОАТ и СЕЛИН, то почти всегда на 3й попытке можно определить ответ. Если нет: ОБДУВ :)

А я с этим не заморачивался, вводил просто рандомное слово в начале, обычно чтобы все буквы были разными, всё равно угадывал за 3-4 попытки обычно) Но я думал ещё над тем чтобы слова по популярности отсортировать, а то в этом списке большая часть слов каких-то специфичных, которые впервые вижу. И среди них потом приходится искать нормальные слова)) Потому что обычно в 5букв загадываются популярные слова, никаких "тамга", "ошкуй", "плебс" там к счастью нет :D

**Comment by mirmozavr** [comment]
@MRGRD56 Я может тоже выложу свой решатель.
После кроата и селина осталось слов 6 подходящих, среди них был ДОБОР, тут я понял, что не хватает добра :)

**Comment by Franek-82** [comment]
Подскажите, как можно из этого списка существительных получить примерно 10 тыс. наиболее распространённых?

**Comment by MRGRD56** [comment]
@Franek-82 можно скачать CSV-версию частотного словаря отсюда  
http://dict.ruslang.ru/freq.php

и с помощью Excel или чего-то подобного, отфильтровать существительные (`PoS=s`), и отсортировать их по частоте использования (`Freq(ipm)`)

если надо, можно отобрать из того словаря только слова, которые есть в этом репозитории (с помощью какой-нибудь СУБД, языка программирования или Excel того же)

**Comment by Franek-82** [comment]
Спасибо! Отфильтрую при помощи Python.
